.gitpod/setup-ddev.sh

```diff
@@ -4,13 +4,13 @@
 
 set -eu -o pipefail
 
-MYDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+DDEV_DIR="$GITPOD_REPO_ROOT/.ddev"
```

Comment:
This is a nice approach, but doesn't work if ddev isn't in the repo root. So... ddev's gitpod integration downloads this script, and there the drupal instance is *not* in the GITHUB_REPO_ROOT. https://github.com/drud/ddev/blob/235ff4b85d03540f4ad080d9c91a89f1f6e8f246/.gitpod.yml#L9

So we'll need to either come up with a generic approach to this or split off the ddev usage.

Reply:
So, are you suggesting that the directory we're running the script from, is <current_directory>, which will keep it flexible?

Reply:
I'm saying that we have to have a way to do it that works in more places than plain-vanilla ddev-gitpod (single project gitpod instance). Maybe an argument to the script?